metasyn/var.py

```diff
@@ -197,13 +139,14 @@ def __str__(self) -> str:
             f'- Distribution:\n{distribution_formatted}\n'
         )
 
-    def fit(self,  # pylint: disable=too-many-arguments
-            dist: Optional[Union[str, BaseDistribution, type]] = None,
-            dist_providers: Union[str, type,
-                                  BaseDistributionProvider] = "builtin",
+    @classmethod
+    def fit(cls,  # pylint: disable=too-many-arguments
+            series: Union[pl.Series, pd.Series],
```

Comment:
again pd.Series support is nice but it would be nice to not have to depend on pd

Reply:
will do this as part of #230